generate/generate_test.go

```diff
@@ -58,11 +58,21 @@ func tmpl(t *testing.T) string {
 }
 
 func TestGenerate(t *testing.T) {
-	require.Equal(t, exp, tmpl(t))
 
+	t.Run("Should be same at least once, in a while", func(t *testing.T) {
```

Comment:
Hmm, this validates what I've found in the past. There's definitely some non-determinism in the parser that needs to be fixed at some point.

Reply:
:+1: :eyeglasses: :grey_question: